src/ResourceManager/TrafficManager/Commands.TrafficManager2/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.TrafficManager" version="2.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.TrafficManager" version="2.5.0" targetFramework="net452" />
```

Comment:
You will need to remove the changes to tools/AzureRM/AzureRM.psm1 in the file below this one.

Reply:
Done.